Bottom: 25e90c9aabf44874e792671f1fff8a64da39008b
Top:    4d0d9bc451ab50d659c9d5e2135416d4296613ce
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-11 16:29:54 -0500

Move ansible role vars to role defaults

This allows other parts of the playbook i.e. host vars, command line the
ability to override these defaults when necessary.


---
diff --git a/roles/docker-openssl/vars/main.yml b/roles/docker-openssl/defaults/main.yml
similarity index 100%
rename from roles/docker-openssl/vars/main.yml
rename to roles/docker-openssl/defaults/main.yml
